src/content/docs/infrastructure/prometheus-integrations/integrations-list/ingress-controller-integration.mdx

```diff
@@ -0,0 +1,79 @@
+---
+title: 'Ingress controller integration'
```

Comment:
I would add nginx in the title of the page

Reply:
I've modified it and now it's 'NGINX Ingress Controller'